wrapper/src/libvmaf.h

```diff
@@ -32,92 +34,87 @@
 extern "C" {
 #endif
 
-int compute_vmaf(double* vmaf_score, char* fmt, int width, int height, int (*read_frame)(float *ref_data, float *main_data, float *temp_data, int stride_byte, void *user_data),
-				 void *user_data, char *model_path, char *log_path, char *log_fmt, int disable_clip, int disable_avx, int enable_transform, int phone_model, int do_psnr,
-				 int do_ssim, int do_ms_ssim, char *pool_method, int n_thread, int n_subsample, int enable_conf_interval);
+#include <stdbool.h>
 
-#ifdef __cplusplus
-}
-#endif
+enum VmafLogFmt {
+    VMAF_LOG_FMT_XML  = (1 << 0),
+    VMAF_LOG_FMT_JSON = (1 << 1),
+    VMAF_LOG_FMT_CSV  = (1 << 2),
+};
```

Comment:
Don't bitshift these enum values unless you intend for them to be or'd and used as a bitfield.

Reply:
Fixed